R/stats-lm-tidiers.R

```diff
@@ -219,33 +218,11 @@ glance.lm <- function(x, ...) {
   )
 }
 
-# compute confidence intervals for mlm object. 
-confint.mlm <- function (object, level = 0.95, ...) {
```

Comment:
Just a note: `confint.mlm()` is new in R 3.5. So removing this means we'll get some error on older versions of R that we should catch / do something informative with. i.e. I think deleting this code is the right way to go, but we should point users who want this functionality toward newer versions of R.

Reply:
Hmm, not sure what to do here. Should there be a conditional statement such that if R version is > `3.5`, then it uses `stats::confint` and otherwise uses this custom function? 

I know `R.version` gives `major` (`R.version$major`) and `minor` (`R.version$minor`) version of `R`, but not sure how to get both of them in a string.

Reply:
Let's `tryCatch()` the `confint()` call that is currently in `process_mlm()` then, with a warning about R versions when that fails.